qualtran/bloqs/basic_gates.ipynb

```diff
@@ -418,9 +418,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from qualtran.resource_counting import get_bloq_counts_graph, GraphvizCounts\n",
+    "from qualtran.resource_counting import get_bloq_counts_graph\n",
+    "from qualtran.drawing import show_counts_graph, show_counts_sigma\n",
+    "\n",
     "g, sigma = get_bloq_counts_graph(bloq)\n",
-    "GraphvizCounts(g).get_svg()"
+    "show_counts_graph(g)\n",
+    "show_counts_sigma(sigma)"
```

Comment:
Can we have a single helper to show both `counts_graph` and `counts_sigma` ? It would be nice to have a single statement, maybe like `show_bloq_counts(counts)`, that displays both of these quantities instead of the current 3 line workflow where we first get the graph and sigma and then display each of them separately.

Reply:
I don't want to merge the count-graph-getting function with the showing function; but I don't think that's what you're proposing.

I _do_ want to put the count-graph-getting function as a method on `Bloq` cc #373 

showing two things with one line? maybe. Can we punt to later?